kong/plugins/acme/client.lua

```diff
@@ -54,6 +54,47 @@ local function new_storage_adapter(conf)
   return storage, st, err
 end
 
+-- return true if domain exists in acme_domain config, else false.
```

Comment:
isn't it possible to refer existing code for both check in config and delete certs
- https://github.com/Instamojo/kong-plugin-acme/blob/master/kong/plugins/acme/handler.lua#L108
- https://github.com/Instamojo/kong-plugin-acme/blob/master/kong/plugins/acme/api.lua#L97

Reply:
doing it

Reply:
doing it